### ISSUE TITLE
Clicking "verify this session" toast on an existing device doesn't re-trigger toasts on other devices.

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#13186]
I just refreshed Riot (having upgraded to 1.6.0-rc1) and got a nice "verify this session" toast.  I clicked on it, and it said "verify by another device".

However, my other devices didn't pop up an "unverified device" warning, because it wasn't a fresh login, and i'd already hit 'later' on the toasts earlier on.  I'd expect the toasts to be retriggered somehow on the other devices otherwise the UX leaves me hanging.

**Comment by ara4n** [comment]
Amandine characterised this as:

> I think the weirdest is "this is an untrusted session use another device to verify it" but nowhere it tells you how, and given our awful path (go to an e2e encrypted room to display your own profile, determine which of the Mobile is the one you are trying to verify , then click on 'untrusted') I'm expecting complaints here

**Comment by jryans** [comment]
On web, we're listening all the time for new devices in the background and trigger toasts when one appears. Perhaps that's missing from mobile...?

**Comment by jryans** [comment]
Sounds like both RiotX and iOS do show toasts on new devices after login, but it can take a while to appear if sync etc. is slow on the server.

In that case, I don't really see what clients could do about it quickly before release, so moving to post-release.

**Comment by ara4n** [comment]
> On web, we're listening all the time for new devices in the background and trigger toasts when one appears. 

The problem is that it's not a new device.  It was an existing device where you're trying to verify it later.

**Comment by nadonomy** [comment]
> I'd expect the toasts to be retriggered somehow on the other devices otherwise the UX leaves me hanging.

+1000 to this being the desired behaviour.

**Comment by thallada** [comment]
Just want to add that this happened to me even when I didn't press "later".

I logged into riot on my windows PC, successfully got the verification notice on my android device, clicked verify on both devices after comparing the emojis, and then I got a loading spinner on the desktop that was lasting forever. Something clearly broke so I exited everything and tried again but I could not verify anymore because my phone was not prompting the verification notification despite me clicking "verify" over and over again on my desktop.

This has happened to me every time single time I log into a new device. I just use the recovery passphrase now.
